modules/monitoring-configuring-cluster-for-application-monitoring.adoc

```diff
@@ -13,6 +13,6 @@ Before application developers can monitor their applications, the human operator
 
 .Procedure
 
-. In the {product-title} web console, navigate to *Operators* -> *OperatorHub* and install the Prometheus Operator in the namespace where your application is.
+. In the {product-title} web console, navigate to the *Operators* -> *OperatorHub* page and install the Prometheus Operator in the namespace where your application is.
 
-. Navigate to *Catalog* -> *Developer Catalog* and install Prometheus, Alertmanager, Prometheus Rule, and Service Monitor in the same namespace.
+. Navigate to the *Catalog* -> *Developer Catalog* page and install Prometheus, Alertmanager, Prometheus Rule, and Service Monitor in the same namespace.
```

Comment:
Navigate to the *Catalog* -> *Developer Catalog* page
It is better change to
Navigate to the *Operators* -> *Installed Operators* page

reason: there is not link in the left navigation area links to /catalog page

Reply:
Implemented in #16978.